apps/admin/frontend/src/components/navigation_screen.tsx

```diff
@@ -55,10 +55,7 @@ const ELECTION_MANAGER_NAV_ITEMS: readonly NavItem[] = _.compact([
     BooleanEnvironmentVariableName.WRITE_IN_ADJUDICATION
   ) && { label: 'Write-Ins', routerPath: routerPaths.writeIns },
   { label: 'Reports', routerPath: routerPaths.reports },
-  isFeatureFlagEnabled(BooleanEnvironmentVariableName.LIVECHECK) && {
-    label: 'System',
-    routerPath: routerPaths.system,
-  },
```

Comment:
LiveCheck removal intentional?

Reply:
The LiveCheck logic is already in the settings screen that I replaced this with. So it's just consolidated now